x-pack/plugins/security_solution/public/management/components/console/console.tsx

```diff
@@ -104,61 +104,61 @@ export const Console = memo<ConsoleProps>(
       }
     }, []);
 
+    // When the console is shown, set focus to it so that user can just start typing
+    useEffect(() => {
+      if (!managedConsole || managedConsole.isOpen) {
+        setTimeout(handleConsoleClick, 2);
```

Comment:
why setTimeout of 2 and not just calling .focus() on the ref directly?

Reply:
in my testing, I need `setTimeout()`. I will check this again in my next PR since I have re-designed how the Console is shown/hidden.